runtimes/runtimes/util/standalone/getProxySettings/parseScutil.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+const TYPE_KEY = '__scutil__type__'
+
+// Quick hacky parser, which translates output into valid JSON:
+export function parseScutilOutput(output: string): {} {
```

Comment:
can you find a library that does this for you? This JSON string manipulation is fragile

Reply:
Discussed offline